sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/implementation/ConsistencyTests2.java

```diff
@@ -6,6 +6,7 @@
 import com.azure.cosmos.BridgeInternal;
```

Comment:
nit: Unrelated to your PR but this file has some unused imports.

Reply:
Fixed